src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -68,74 +68,74 @@ public void setGuiSettings(GuiSettings guiSettings) {
     }
 
     @Override
-    public Path getAddressBookFilePath() {
-        return userPrefs.getAddressBookFilePath();
+    public Path getLibTaskFilePath() {
+        return userPrefs.getLibTaskFilePath();
     }
 
     @Override
-    public void setAddressBookFilePath(Path addressBookFilePath) {
-        requireNonNull(addressBookFilePath);
-        userPrefs.setAddressBookFilePath(addressBookFilePath);
+    public void setLibTaskFilePath(Path libTaskFilePath) {
+        requireNonNull(libTaskFilePath);
+        userPrefs.setLibTaskFilePath(libTaskFilePath);
     }
 
-    //=========== AddressBook ================================================================================
+    //=========== LibTask ================================================================================
 
     @Override
-    public void setAddressBook(ReadOnlyAddressBook addressBook) {
-        this.addressBook.resetData(addressBook);
+    public void setLibTask(ReadOnlyLibTask libTask) {
+        this.libTask.resetData(libTask);
     }
 
     @Override
-    public ReadOnlyAddressBook getAddressBook() {
-        return addressBook;
+    public ReadOnlyLibTask getLibTask() {
+        return libTask;
     }
 
     @Override
     public boolean hasPatron(Patron patron) {
         requireNonNull(patron);
-        return addressBook.hasPatron(patron);
+        return libTask.hasPatron(patron);
     }
 
     @Override
     public boolean hasBook(Book book) {
         requireNonNull(book);
-        return addressBook.hasBook(book);
+        return libTask.hasBook(book);
     }
 
     @Override
     public void deletePatron(Patron target) {
-        addressBook.removePatron(target);
+        libTask.removePatron(target);
     }
 
     @Override
     public void deleteBook(Book target) {
-        addressBook.removeBook(target);
+        libTask.removeBook(target);
     }
 
     @Override
     public void addPatron(Patron patron) {
-        addressBook.addPatron(patron);
+        libTask.addPatron(patron);
         updateFilteredPatronList(PREDICATE_SHOW_ALL_PATRONS);
     }
 
     @Override
     public void addBook(Book book) {
-        addressBook.addBook(book);
+        libTask.addBook(book);
         updateFilteredBookList(PREDICATE_SHOW_ALL_BOOKS);
     }
 
     @Override
     public void setPatron(Patron target, Patron editedPatron) {
         requireAllNonNull(target, editedPatron);
 
-        addressBook.setPatron(target, editedPatron);
+        libTask.setPatron(target, editedPatron);
     }
 
     @Override
     public void setBook(Book target, Book editedBook) {
         requireAllNonNull(target, editedBook);
 
-        addressBook.setBook(target, editedBook);
+        libTask.setBook(target, editedBook);
     }
 
     //=========== Filtered Patron List Accessors =============================================================
```

Comment:
lines 145 and 154 still has an instance of Addressbook

Reply:
Good catch!

Reply:
Not changing this first, because I think `versionedAddressBook` is some class in AB4, which we don't have. I can't seem to find the class in our project folder for VersionedAddressBook so I don't want to break it yet